### PR TITLE
Initial draft of release notes for 2022-10 update

### PR DESCRIPTION
**Pull request by francine-vendia**

### CODE REVIEW

**Comment by francine-vendia** [comment]
GraphQL_Explorer_AutoFill_ID

File: releases/2022-10-28-releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+draft: false
+tags:
+- Improvements
+- Azure
+- Developer Experience
+---
+
+They're at it again! The Vendia Engineering team has a series of new improvements to continue advancing the experience building on Vendia.  
+
+As always, see our [Docs](https://www.vendia.com/docs/share) for the complete details of each new feature and our [Examples](https://github.com/vendia/examples) for more guidance.
```
Comment (james-vendia) [inline releases/2022-10-28-releasenotes.md:14]:
```suggestion
As always, see [our docs](https://www.vendia.com/docs/share) for the complete details of each new feature and [our examples](https://github.com/vendia/examples) for more guidance. 
```

Comment (james-vendia) [reply releases/2022-10-28-releasenotes.md:14]: Take or leave...

File: releases/2022-10-28-releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- Developer Experience
+---
+
+They're at it again! The Vendia Engineering team has a series of new improvements to continue advancing the experience building on Vendia.  
+
+As always, see our [Docs](https://www.vendia.com/docs/share) for the complete details of each new feature and our [Examples](https://github.com/vendia/examples) for more guidance. 
+Questions? Please [contact us](https://www.vendia.com/contact-us).
+
+# We're COMical 
+While we are quite amusing, we are also officially on [Vendia.com](https://www.vendia.com)! Previously we were nothing-but-net [(Vendia.net)](https://vendia.net/) but now our website is live on Vendia.com. Check it out yourself! Don't worry; development is still on [share.vendia.net](https://share.vendia.net).
```
Comment (james-vendia) [inline releases/2022-10-28-releasenotes.md:18]:
```suggestion
While we are quite amusing, we are also officially on [vendia.com](https://www.vendia.com)! Previously we were nothing-but-net [(vendia.net)](https://vendia.net/) but now our website is live on vendia.com. Check it out yourself! Don't worry; development is still on [share.vendia.net](https://share.vendia.net).
```

File: releases/2022-10-28-releasenotes.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+# Advancing Azure Beta Feature
```
Comment (james-vendia) [inline releases/2022-10-28-releasenotes.md:20]: I'd scrap this section.  Azure is still in beta and it's not clear from the description what's new/different/better.  My general rule is that if we can't show a concrete example, we shouldn't share it in external release notes.

File: releases/2022-10-28-releasenotes.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Sharing, without limitation, is our ethos and we continue expanding the surface area on which to share. We've added validations and stabilization for sharing between two azure-backed nodes by resolving errors associated with metadata availability, handling mutations, and cross-cloud file sharing.
+
+# Ensuring Transaction Success
```
Comment (james-vendia) [inline releases/2022-10-28-releasenotes.md:23]: Can we / are we comfortable quantifying the % increase?  No need to share hard numbers, but can we share improvement %?

File: releases/2022-10-28-releasenotes.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+We continue to make improvements to the speed of [consensus](https://www.vendia.com/docs/share/terms-and-definitions#consensus) and success rate of transactions by reducing scenarios in which errors might display. This release focused on resolving errors due to empty versions of data, complex access rules (ACLs), duplicate attribute names exist in different nested paths, and various uni deployments. We also removed any restrictions caused by WAF API limits.
+
+# Keeping the SDK Current
```
Comment (james-vendia) [inline releases/2022-10-28-releasenotes.md:26]: Example code here would be good.  Both the generated code (where the default is applied) and the approach to override (if the previous default of `Node_Ledgered` is still desired).

**Review by james-vendia — CHANGES_REQUESTED** [review]
A few places to cut, a few places to expand.

Adding in the nested filter example (you can borrow Matt's screenshots from the issue itself) is the big one...and should probably be what we lead with (that and SDK default syncMode change).

**Comment by francine-vendia** [comment]
Filtering_on_Nested_Attribute

File: releases/2022-10-28-releasenotes.md (hunk 1 above)
Comment (francine-vendia) [reply releases/2022-10-28-releasenotes.md]: I updated it in local branch and pushed - should be incorporated now

File: releases/2022-10-28-releasenotes.md (hunk 3 above)
Comment (francine-vendia) [reply releases/2022-10-28-releasenotes.md]: Makes sense - removed.

File: releases/2022-10-28-releasenotes.md (hunk 2 above)
Comment (francine-vendia) [reply releases/2022-10-28-releasenotes.md]: I updated it in local branch and pushed - should be incorporated now

**Comment by francine-vendia** [comment]
@james-vendia - Incorporated the suggested updates and added items from both release-notes-2022-10-27 and the nested filtering. This PR is updated with the latest version.

Note - The `2022-10-27` release had to be rolled back but *may* still get pushed tomorrow. If it does, these can remain as-is. If not, we'll need to remove `# Bug Squashing: Lambda Subscriptions`, the note on `#ref` fix in `# Ensuring Transaction Success` and `# Returning More in Lists`.

Let me know if all looks good and we'll queue it up.

File: releases/2022-10-28-releasenotes.md (hunk 5 above)
Comment (francine-vendia) [reply releases/2022-10-28-releasenotes.md:32]: added!

File: releases/2022-10-28-releasenotes.md (hunk 4 above)
Comment (james-vendia) [reply releases/2022-10-28-releasenotes.md:29]: If it's just error handling, lets cut it.  We usually don't get into defect fixes for the public / non-customer version of release notes.

**Review by james-vendia — APPROVED** [review]
Thanks for the updates.  One additional suggested deletion but otherwise LGTM.

**Comment by james-vendia** [comment]
May want to get a better version of the image here - https://github.com/vendia/blog/pull/224#issuecomment-1294033551.  Quality looks lower than we'd like for public content, unless it's a GH rendering issue.

**Comment by francine-vendia** [comment]
Filtering_on_Nested_Attribute3
New image for nested filtering

**Comment by francine-vendia** [comment]
@james-vendia  - latest round of updates are in based on what was rollbacked vs released (and removed all references to fixes errors / bug squashing). Also updated the image for nested filtering - let me know if that one looks better.

**Comment by james-vendia** [comment]
🚢

**Pull request merged**